app/helpers/question_helper.rb

```diff
@@ -0,0 +1,34 @@
+
+
+module QuestionHelper
+  DAY = 24*360
+
+  def format_asked_time(question)
+    formated_string = ""
+    Time.zone= "Eastern Time (US & Canada)"
+    now = Time.zone.local_to_utc(Time.now)
+
+    interval = (now-question.created_at).to_i
+
+    ##TODO add days
+    if interval > DAY
+      formated_string = (interval/DAY).to_s<< " days"
+    elsif interval > 360
+      formated_string = (interval/360).to_s<< " hours"
+    elsif interval >60
+      formated_string = (interval/60).to_s<< " mins"
+    else
+      formated_string = interval.to_i.to_s<< " secs"
+    end
+    formated_string<<" ago"
+  end
+
+  def get_view_count(question)
+    return 0 if question.view_count == nil
```

Comment:
Why would view_count ever be nil? It might be better to default it to zero then you wouldn't need this helper.